src/components/TutorialTags.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import styled from "styled-components"
+
+import Pill from "./Pill"
+
+// Represent string as 32-bit integer
+const hashCode = (string) => {
+  let hash = 0
+  for (const char of string) {
+    const code = char.charCodeAt(0)
+    hash = (hash << 5) - hash + code
+    hash |= 0
+  }
+  return Math.abs(hash)
+}
+
+// TODO update to Theme variables
```

Comment:
@ryancreatescopy ready for you to sub in the actual colors 🎨 

Please add theme dark & light mode in Theme.js

Reply:
@samajammin not sure how to add theme values into this array

Reply:
![image](https://user-images.githubusercontent.com/40891631/94536169-5ad1f500-023a-11eb-9d7a-35b161edb1d0.png)

Reply:
Looks good @ryancreatescopy! One thought - the 2nd green (mint?) looks very similar to the first green (at least for me), particular on dark mode. I wonder if we replace that with a purple or brown or some other more distinct color?